horovod/run/run.py

```diff
@@ -294,16 +294,21 @@ def parse_args():
     parser.add_argument('-p', '--ssh-port', action="store", dest="ssh_port",
                         type=int, help="SSH port on all the hosts.")
 
-    parser.add_argument('-H', '--host', action="store", dest="host",
-                        help="To specify the list of host names as well as the "
-                             "number of available slots on each host for "
-                             "training processes using the following format: "
-                             "<hostname>:<number of slots>,... . "
-                             "E.g., host1:2,host2:4,host3:1 "
-                             "indicates that 2 processes can run on "
-                             "host1, 4 processes on host2, and 1 process "
-                             "on host3.")
-
+    host_group = parser.add_argument_group("Use one of the following options "
+                                           "to specify which hosts (nodes) of "
+                                           "the cluster to run on")
+    host_group.add_argument('-H', '--host', action="store", dest="host",
+                            help="To specify the list of host names as well "
+                                 "as the number of available slots on each "
+                                 "host for training processes using the "
+                                 "following format: <hostname>:<number of "
+                                 "slots>,... . E.g., host1:2,host2:4,host3:1 "
+                                 "indicates that 2 processes can run on "
+                                 "host1, 4 processes on host2, and 1 process "
+                                 "on host3.")
+    host_group.add_argument('-hostfile', '--hostfile', action="store",
+                            dest="hostfile",
+                            help="Provide a hostfile to use")
```

Comment:
Please change the help section to:

Specifies a host file with the list of hostnames as well as the number of available slots on each host. Each line of the host file is formatted as <hostname>:<number of processes in the host>